code/__defines/items_clothing.dm

```diff
@@ -64,7 +64,7 @@
 #define BLOCKHAIR       0x40    // Hides the user's hair, facial and otherwise.
 
 //This flag applies to gloves, uniforms, shoes, masks, ear items, glasses
-#define ALWAYSDRAW	32//If set, this item is always rendered even if its slot is hidden by other clothing
+#define ALWAYSDRAW	0x80//If set, this item is always rendered even if its slot is hidden by other clothing
```

Comment:
It's interesting that we didn't have a bitflag conflict thus far. Because 32 = 0x20, which should ahve overlapped with BLOCKHEADHAIR.